src/index.js

```diff
@@ -0,0 +1,14 @@
+/* eslint-disable import/no-cycle */
+import './css/styles.css';
+import _ from 'lodash';
+
+function component() {
+  const element = document.createElement('div');
```

Comment:
- [ ] Kindly use the ES6 syntax for function definition (arrow function), as one of the objectives of this project is to practice ES6 syntax. :pray:

Reply:
Recommendations implemented. Thank you